test/d_do_test.d

```diff
@@ -436,7 +440,25 @@ bool compareOutput(string output, string refoutput)
     }
 }
 
+string envGetRequired(in char[] name)
```

Comment:
why not `string`?

Reply:
`const char[]` conveys that the contents of the string do not need to be preserved. If you look at `environment.get` you'll see it also uses `in char[]`.  A function that accepts `string` implies that it requires the contents of that string to remain unchanged for the life of the program, however, the `environment.get` function, and this function do not have this requirement.

Reply:
Well in this case you only call this function with pre-defined strings which are immutable by definition ;-)